packages/asset-kit/src/AssetKitBlock.spec.ct.tsx

```diff
@@ -0,0 +1,365 @@
+            },
+        });
+        mount(<AssetKitBlockWithStubs />);
+        cy.get(BLOCK_SELECTOR).should('have.css', 'padding', '0px');
+    });
+
+    it('should display with padding on block if border is setted', () => {
+        const [AssetKitBlockWithStubs] = withAppBridgeBlockStubs(AssetKitBlock, {
+            blockSettings: {
+                hasBorder_blocks: true,
+            },
+            blockAssets: {
+                [ASSET_SETTINGS_ID]: [AssetDummy.with(1), AssetDummy.with(2)],
+            },
+        });
+        mount(<AssetKitBlockWithStubs />);
+        cy.get(BLOCK_SELECTOR).should('have.css', 'padding', '32px');
+    });
+
+    it('should display with padding on block if background is setted', () => {
+        const [AssetKitBlockWithStubs] = withAppBridgeBlockStubs(AssetKitBlock, {
+            blockSettings: {
+                hasBackground_blocks: true,
+            },
+            blockAssets: {
+                [ASSET_SETTINGS_ID]: [AssetDummy.with(1), AssetDummy.with(2)],
+            },
+        });
+        mount(<AssetKitBlockWithStubs />);
+        cy.get(BLOCK_SELECTOR).should('have.css', 'padding', '32px');
+    });
+
+    it('should display with padding on block if background and border is setted', () => {
+        const [AssetKitBlockWithStubs] = withAppBridgeBlockStubs(AssetKitBlock, {
+            blockSettings: {
+                hasBackground_blocks: true,
+                hasBorder_blocks: true,
+            },
+            blockAssets: {
+                [ASSET_SETTINGS_ID]: [AssetDummy.with(1), AssetDummy.with(2)],
+            },
+        });
+        mount(<AssetKitBlockWithStubs />);
+        cy.get(BLOCK_SELECTOR).should('have.css', 'padding', '32px');
+    });
+
+    it('should display block with black background color', () => {
+        const [AssetKitBlockWithStubs] = withAppBridgeBlockStubs(AssetKitBlock, {
+            blockSettings: {
+                hasBackground_blocks: true,
+                backgroundColor_blocks: BLACK,
+            },
+            blockAssets: {
+                [ASSET_SETTINGS_ID]: [AssetDummy.with(1), AssetDummy.with(2)],
+            },
+        });
+        mount(<AssetKitBlockWithStubs />);
+        cy.get(BLOCK_SELECTOR).should('have.css', 'background-color', 'rgb(0, 0, 0)');
+    });
+
+    it('should display block with white background color', () => {
+        const [AssetKitBlockWithStubs] = withAppBridgeBlockStubs(AssetKitBlock, {
+            blockSettings: {
+                hasBackground_blocks: true,
+                backgroundColor_blocks: WHITE,
+            },
+            blockAssets: {
+                [ASSET_SETTINGS_ID]: [AssetDummy.with(1), AssetDummy.with(2)],
+            },
+        });
+        mount(<AssetKitBlockWithStubs />);
+        cy.get(BLOCK_SELECTOR).should('have.css', 'background-color', 'rgb(255, 255, 255)');
+    });
+
+    it('should display block with border 2px dashed black', () => {
```

Comment:
I think 1 test with a border & border radius is enough. Currently there are 3 test cases which are really similar.